CondTools/Hcal/src/HcalPFCutsHandler.cc

```diff
@@ -0,0 +1,36 @@
+#include "CondTools/Hcal/interface/HcalPFCutsHandler.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/HcalDetId/interface/HcalGenericDetId.h"
+#include <memory>
+
+HcalPFCutsHandler::HcalPFCutsHandler(edm::ParameterSet const& ps) {
+  m_name = ps.getUntrackedParameter<std::string>("name", "HcalPFCutsHandler");
+  sinceTime = ps.getUntrackedParameter<unsigned>("IOVRun", 0);
+}
+
+HcalPFCutsHandler::~HcalPFCutsHandler() {}
+
+void HcalPFCutsHandler::getNewObjects() {
+  //  edm::LogInfo   ("HcalPFCutsHandler")
+  std::cout << "------- " << m_name << " - > getNewObjects\n"
```

Comment:
please use the LogInfo (or LogPrint etc) instead of teh `cout`, you also dont need `std::endl` when you use the MsgLogger functions

Reply:
Again, this is just a boilerplate, with 41 pre-existing *Handler.cc in CondTools/Hcal/src out of which 35 print to cout. Not sure why this one should be different.

Reply:
`couts` are strictly forbidden for new developments, so please change those, the rest of the package can be updated later

Reply:
OK, prints to cout are removed in HcalPFCutsHandler.cc